_questions/en/1-waarom-is-coronamelder-nodig.md

```diff
@@ -9,6 +9,10 @@ categories:
 - faq
 showBreadCrumbs: true
 showContact: true
+featuredHome: true
+featuredHomeIndex: 1
```

Comment:
Zouden we scherp kunnen kijken naar de andere waarden in de Frontmatter? Om ruis/onduidelijkheid te voorkomen voor de content manager.

Reply:
@anneke Lijkt mij een goed plan. Maar denk ik wel beter om hiervoor een nieuwe PR van te maken.